.pyenchant_pylint_custom_dict.txt

```diff
@@ -0,0 +1,367 @@
+abc
+abcmeta
+abstractproperty
+analyse
+analysed
+analysing
+arg
+argparse
+args
+argumentdefaultshelpformatter
+argumentparser
+argumentsparser
+argv
+ascii
+assignattr
+assignname
+ast
+AST
+astroid
+async
+asynccontextmanager
+attr
+attrib
+autogenerated
+backend
+backticks
+BaseChecker
+basename
+behaviour
+bidi
+bla
+bom
+bool
+boolean
+boolop
+boundmethod
+builtins
+bw
+callables
+cardinality
```

Comment:
This makes me wonder how up to date the base dictionary is..

Reply:
Is there a `EN_US/EN_GB` dictionary? That combines the two?

Reply:
Turns out that there might be a simple solution : https://stackoverflow.com/a/69540620/2519059

Reply:
```diff
diff --git a/.pyenchant_pylint_custom_dict.txt b/.pyenchant_pylint_custom_dict.txt
index e4a1f8f06..67086db8c 100644
--- a/.pyenchant_pylint_custom_dict.txt
+++ b/.pyenchant_pylint_custom_dict.txt
@@ -26,7 +26,6 @@ backend
 backticks
 BaseChecker
 basename
-behaviour
 bidi
 bla
 bom
diff --git a/pylintrc b/pylintrc
index dbe919b47..c45d9f38d 100644
--- a/pylintrc
+++ b/pylintrc
@@ -392,7 +392,7 @@ missing-member-max-choices=1
 
 # Spelling dictionary name. Available dictionaries: none. To make it working
 # install python-enchant package.
-spelling-dict=en_US
+spelling-dict=en
 
 # List of comma separated words that should not be checked.
 spelling-ignore-words=
```
This does not work sadly. I think we'll keep our little private dict up to date it's the simple way to fix this.

Reply:
Hm, I was reading the read me of pyenchant and it actually says it's quite flexible in handling multiple languages.

I'd be willing to spend some time to look whether we can make the dictionary option a csv (which I think is backwards compatible) and then allow the usage of multiple.

Reply:
> I'd be willing to spend some time to look whether we can make the dictionary option a csv (which I think is backwards compatible) and then allow the usage of multiple.

I'll check the windows tests failing, and there's a lot of MR in the pipe right now feel free to update this one :)

Reply:
Well nvm, the windows test magically got fixed, probably thanks to #6459. Would you mind doing a follow-up ? Because every MR that is not spell checked add new issues.